### DISCUSSION TITLE
Healthcheck endpoint ?

### DISCUSSION THREAD

**Discussion by rgarrigue** [browserless/chrome#3905] [Q&A] [answered]
Hello,

I saw no clear answer on what I should check for browserless v2 healthcheck endpoint ? For kubernetes readiness & liveness probes

Thanks

**Comment by joelgriffith** [comment, marked as answer]
https://github.com/browserless/browserless/pull/3933 introduces an /active endpoint that allows for readiness probing, and somewhat for liveliness as well

**Reply by jenky** [reply]
Sorry to bring this up again. Is there any way to call the endpoint without specifying the token?

**Comment by introvert** [comment]
why would you require the token on the health endpoint?
